CRCTray.cs

```diff
@@ -77,9 +77,8 @@ private void SetContextMenu()
             cm.AccessibleName = "menu";
 
             // Status Menu
-            ToolStripLabel status = new ToolStripLabel(InitialState); //TODO: actually "Unknown"
-            status.AccessibleName = status.Text;
-            status.TextChanged += Status_TextChanged;

```

Comment:
Do we need to remove this too?  `status.TextChanged += Status_TextChanged;` the fix only removes the local variable